packages/cli/generation/remote-generation/remote-workspace-runner/src/publishDocs.ts

```diff
@@ -162,18 +167,81 @@ export async function publishDocs({
         },
         async ({ ir, snippetsConfig, playgroundConfig, apiName }) => {
             const apiDefinition = convertIrToFdrApi({ ir, snippetsConfig, playgroundConfig, context });
+            
+            let dynamicIr: Record<string, DynamicIntermediateRepresentation> = {};
+            const languages: GeneratorLanguage[] = ["typescript", "python"];
+            for (const language of languages) {
+                try {
+                    // generate language-specific dynamic irs
+                    dynamicIr[language] = convertIrToDynamicSnippetsIr({
+                        ir,
+                        disableExamples: true,
+                        generationLanguage: language
+                    });
+                } catch (error) {
+                    context.logger.debug(`Failed to create dynamic IR for ${language}: ${error}`);
```

Comment:
If there are more than 1 APIs, you also want the name of the API in this log message